CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 * Upgrade to pulumi-terraform-bridge v2.23.0
+* Add support for enabling encryptionConfig as part of aws.eks.Cluster
```

Comment:
It might be worth a note that this takes a _very_ long time to apply - 50 minutes in my testing.